tools/cli/adminFailoverCommands.go

```diff
@@ -281,3 +271,53 @@ func getOperator() string {
 
 	return fmt.Sprintf("%s (username: %s)", user.Name, user.Username)
 }
+
+func isWorkflowTerminated(descResp *shared.DescribeWorkflowExecutionResponse) bool {
+	return descResp.WorkflowExecutionInfo.CloseStatus != nil &&
+		descResp.WorkflowExecutionInfo.CloseStatus.Equals(shared.WorkflowExecutionCloseStatusTerminated)
+}
+
+func executePauseOrResume(c *cli.Context, isPause bool) error {
+	client := getCadenceClient(c)
+	tcCtx, cancel := newContext(c)
+	defer cancel()
+
+	workflowID := getWorkflowID(c)
+	runID := getRunID(c)
+	var signalName string
+	if isPause {
+		signalName = failovermanager.PauseSignal
+	} else {
+		signalName = failovermanager.ResumeSignal
+	}
+
+	return client.SignalWorkflow(tcCtx, workflowID, runID, signalName, nil)
+}
+
+func validateStartParams(params *startParams) {
+	if len(params.targetCluster) == 0 {
+		ErrorAndExit("targetCluster is not provided", nil)
+	}
+	if len(params.sourceCluster) == 0 {
+		ErrorAndExit("sourceCluster is not provided", nil)
+	}
+	if params.targetCluster == params.sourceCluster {
+		ErrorAndExit("targetCluster is same as sourceCluster", nil)
+	}
+	if params.batchFailoverSize <= 0 {
+		params.batchFailoverSize = defaultBatchFailoverSize
+	}
+	if params.batchFailoverWaitTimeInSeconds <= 0 {
+		params.batchFailoverWaitTimeInSeconds = defaultBatchFailoverWaitTimeInSeconds
+	}
+	if params.failoverTimeout <= 0 {
+		params.failoverTimeout = defaultFailoverTimeoutInSeconds
+	}
+}
+
+func validateWorkflowID(workflowID string) string {
```

Comment:
Maybe I missed something, but I don't see this function used anywhere?

Reply:
Yes. This should be removed